docs/common/help.go

```diff
@@ -21,6 +21,19 @@ const GlobalEnvVars string = `	JFROG_CLI_LOG_LEVEL
 		[Default: The operating system's temp directory]
 		Defines the temp directory used by JFrog CLI.
 
+	JFROG_CLI_BUILD_NAME
+		Build name to use in build related commands. This environment variable will be used in case of omitted --build-name flag and <build name> argument.
```

Comment:
I suggest rephrasing this description to:
Build name to be used by commands which expect a build name, unless sent as a command argument or option.